package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "@types/mediawiki",
+	"version": "0.1.0",
+	"description": "TypeScript definitions of MediaWiki interfaces",
+	"scripts": {
+		"build:types": "tsc -p tsconfig.json",
+		"build": "npm-run-all build:*",
+		"test:lint": "eslint .",
+		"test": "npm-run-all test:*"
+	},
+	"license": "GPL-2.0-or-later",
```

Comment:
I haven't published a types-only package yet but in another repo, I had to explicitly add the types like: `"types": "dist/index.d.ts"`.

Reply:
I believe it is needed, added this.
It also looks that the directory structure is too complex. And what exactly is exported should be revised. I am not sure there is much use for `MediaWiki` interface, as it is very incomplete. Exposing `MwLog` to the consumers might be a good start. I am a TypeScript noob, I'd need to play with a bit more.